index.html

```diff
@@ -311,28 +311,33 @@ <h2>Reporting</h2>
 
     <p>You may instead choose to raise issues with a chair of the group. A chair generally knows the context of the group
       so may be better able to assist you. If they do not feel able to assist you, they should help you raise
-      the issue with an Ombuds person. You may also ask for them to help you raise it to an Ombuds person immediately
+      the issue with an Ombuds. You may also ask for them to help you raise it to an Ombuds immediately
       if you do not feel comfortable contacting the Ombuds directly. 
     </p>
 
     <p>If you are in a <a>leadership position</a> and you witness harassment or other 
       <a href="#unacceptablebehavior">unacceptable behaviour</a> you should raise issues directly with
-      an <a href="https://www.w3.org/Consortium/pwe/#ombuds">Ombuds</a> person who can assist you.
+      an <a href="https://www.w3.org/Consortium/pwe/#ombuds">Ombuds</a> who can assist you.
     </p>
     
     <p>Any person in a <a>leadership position</a> should provide immediate assistance to any participant
       who wants to contact an ombuds. Failure to do so may increase the harmful consequences of the action being reported.
     </p>
     
     <p>Chairs, Team Contacts and Event Organisers should take such immediate action as they deem necessary in order to stop
-      harmful effects of harassment, bullying, or other <a href="#unacceptablebehaviour">unacceptable behaviour</a>.
-      This action may take many forms, but examples include:
+      <a href="#unacceptablebehaviour">unacceptable behaviour</a>. This action may take many forms, but examples may include:
     </p>
-    
+    <h3>Immediately:</h3>
+    <ul>
+      <li>Point out if someone is violating the CEPC, to give them the chance to withdraw or edit their statement.</li>
+      <li>Remind participants that meetings & work operate under the CEPC.</li>
+      <li>Asking someone to leave a meeting, or a conversation thread.</li>
+    </ul>
+    <h3>After the meeting:</h3>
     <ul>
-      <li>ejecting someone from a meeting or event</li>
-      <li>Blocking them on a mailing list</li>
-      <li>Removing specific comments from an archive, or the harmful parts of such comments</li>
+      <li>Follow up with affected participants, this may be seperate meetings.</li>
+      <li>The Chair may reach out to an Ombuds for assistance</li>
+      <li>If the Chair needs assistance, they can make use of the Chairs Training program.</li>
```

Comment:
If I need assistance in this situation I would not want to go to a training programme, I would want to go to a person. Suggest rewording to:

>Further information and resources for Chairs are available via the Chairs Training program.

Reply:
agree - and move it out of the list to be a following paragraph